src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java

```diff
@@ -215,8 +216,14 @@ private void processZipFile(
     }
 
     private ZipInputStream loadIntoMemory(CloudBlockBlob cloudBlockBlob, String zipFilename) throws StorageException {
+        log.info("Loading {} into memory.", zipFilename);
```

Comment:
How about `Loading file {} into memory.`? Easier to search.

Reply:
Same with the other log

Reply:
fixed